src/pr-info.ts

```diff
@@ -399,11 +398,17 @@ configSuspicious["OTHER_FILES.txt"] = contents =>
     // all path parts are not empty and not all dots
     : !contents.split(/\n/).every(line => line === "" || isRelativePath(line)) ? "bad path"
     : undefined;
-configSuspicious["tslint.json"] = contents =>
+configSuspicious["tslint.json"] = (contents, oldText) => {
     // just the required form
-    JSON.stringify(JSON.parse(contents)) !== JSON.stringify({ "extends": "dtslint/dt.json" })
-    ? "not the required form"
-    : undefined;
+    try {
```

Comment:
Since this is similar to the other piece of code I'm tempted to extract that into a function, but fine to leave it as is for now.

Reply:
I'm unopposed, I just haven't tackled this.